Frontend/ui-driver/src/Flow.purs

```diff
@@ -444,13 +445,32 @@ getDriverInfoFlow event activeRideResp driverInfoResp updateShowSubscription isA
           let cityConfig = getCityConfig config.cityConfig (getValueToLocalStore DRIVER_LOCATION)
           when (updateFeatureFlags && cityConfig.callDriverInfoPost) $ do
             void $ lift $ lift $ fork $ Remote.getDriverInfoApi ""
+  --        let _ = spy "rahul-driverInfoResp" getDriverInfoResp
           if getDriverInfoResp.enabled 
             then do
               deleteValueFromLocalStore ENTERED_RC
               if getValueToLocalStore IS_DRIVER_ENABLED == "false" 
                 then do
                   void $ pure $ firebaseLogEvent "ny_driver_enabled"
                   void $ pure $ metaLogEvent "ny_driver_enabled"
+                  case getDriverInfoResp.linkedVehicle of 
+                    Just linkedVehicle -> case linkedVehicle of 
+                        (Vehicle value) -> case value.category of 
```

Comment:
Can we remove this many case matches and use fromMaybe like:
`let (Vehicle linkedVehicle) = fromMaybe dummyVehicleObject getDriverInfoResp.linkedVehicle`